tfjs-core/src/ops/browser.ts

```diff
@@ -154,6 +154,68 @@ function fromPixels_(
   return tensor3d(values, outShape, 'int32');
 }
 
+/**
+ * Creates a `tf.Tensor` from an image in async way.
+ *
+ * ```js
+ * const image = new ImageData(1, 1);
+ * image.data[0] = 100;
+ * image.data[1] = 150;
+ * image.data[2] = 200;
+ * image.data[3] = 255;
+ *
+ * await tf.browser.fromPixelsAsync(image).print();
```

Comment:
Does it work if using `await tf.browser.fromPixelsAsync(image).print()`? Please add some tests in `from_pixels_test.ts`.